common/cb_types/31_pony.txt

```diff
@@ -2,7 +2,7 @@
 cb_overseas = {
 	prerequisites = {
 		cb_on_overseas = yes
-		technology_group = pony
+		technology_group = enlightened_pony
```

Comment:
I thought I'd rename them enlightened_pony, rather than just pony, in case pony in the future could refer to the regular unenlightened kind. I dunno.